src/util/ConfigUtils.ts

```diff
@@ -168,3 +184,131 @@ export function modifyModulesNotMatchingTargetVersion(
 	}
 	return oModifiedConfig;
 }
+
+/**
+ * Merges the same modules with different versions
+ *
+ * Before:
+ * "modules": {
+ *		"jquery.sap.script": {
+ *			"jQuery.sap.extend@1.58.0": {
```

Comment:
Should be documented that the version in the key is redundant to the version property due to compatibility reasons.

Reply:
done